### PR TITLE
Fix Email Performance

### PR DESCRIPTION
**Pull request by james-hollinger**
Properly cache email templates rather than compiling and caching every time.

### CODE REVIEW

**Review by johnnyl6 — APPROVED** [review]
LGTM

**Review by AMiKaH — APPROVED** [review]

**Review by neophyte57 — CHANGES_REQUESTED** [review]
I setup some data and currently running send-renewal-emails cron job against this PR.  If web-api pod restarts a lot less than other PR's pod did, by EOD, then things look good

File: prime-dotnet-webapi/Services/EmailRenderingService.cs [hunk 1]
```diff
@@ -15,14 +15,10 @@ public class EmailRenderingService : IEmailRenderingService
         private const string MohEmail = "HLTH.HnetConnection@gov.bc.ca";
         private const string ProviderEnrolmentTeamEmail = "Lori.Haggstrom@gov.bc.ca";
```
Comment (neophyte57) [inline prime-dotnet-webapi/Services/EmailRenderingService.cs:16]: While this isn't part of the PR changes, shouldn't this value be in a ConfigMap, that is, environment variable?  What happens when Lori Haggstrom retires or leaves?

**Comment by neophyte57** [comment]
> I setup some data and currently running send-renewal-emails cron job against this PR. If web-api pod restarts a lot less than other PR's pod did, by EOD, then things look good

The web-api pod is doing very well.  No restarts despite sending out emails every 15 minutes.  Other PR was restarting about once per hour.  Ready to test in `dev` environment.

**Review by neophyte57 — APPROVED** [review]
Looks good except that comment about hard-coded email address

**Review by hogmann — APPROVED** [review]
L G T M

File: prime-dotnet-webapi/Services/EmailRenderingService.cs (hunk 1 above)
Comment (james-hollinger) [reply prime-dotnet-webapi/Services/EmailRenderingService.cs:16]: Could be a good idea, but having those emails be configurable has never been a requirement

Comment (neophyte57) [reply prime-dotnet-webapi/Services/EmailRenderingService.cs:16]: Worthwhile for Product Owner to consider?

Comment (itchison) [reply prime-dotnet-webapi/Services/EmailRenderingService.cs:16]: @neophyte57 Can you create a ticket for this comment and then resolve this conversation.

**Review by itchison — APPROVED** [review]
SHIP IT

**Pull request merged**